apps/vr-tests-react-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluentui/vr-tests-react-components",
-  "version": "1.0.0",
+  "version": "9.0.0-rc.0",
```

Comment:
why do we need to bump the actual applicaton ? that doesn't make any sense to me ? 

**A: @ling1726 **

- syncpack screams
- we could alternatively create a separate version group for vr-tests to make syncpack happy
- or use * as dependenceis for theapp
- we already use * for deps like eslint-plugin to avoid the syncpack screaming
- or even better - update beachball to handle publishing 2 different libraries out of a repo property

Reply:
@ling1726  yeah storybook just aliases everything

Reply:
kk so lets do it

Reply:
can do so in a follow up, this PR is just to unblock CI